src/statsig/index.ts

```diff
@@ -52,3 +56,14 @@ export function getDynamicConfigParams<T extends Record<string, StatsigParameter
     return defaultValues
   }
 }
+
+export async function updateStatsigUser(statsigUser?: StatsigUser) {
+  const state = store.getState()
```

Comment:
I wasn't sure of the correct pattern to use to get state from the store outside a component. We use this pattern in a few other places that I saw, so copied it here, but open to other suggestions!